tictactoe/tictactoe-algorithm.md

```diff
@@ -0,0 +1,6 @@
+# tictactoe algorithm
+	1.Initialize the 3x3 array using the list.
+	2.Introduce first player is X and second player is O and ask user to select.
```

Comment:
* Check every cell for any peg present or not and display a message peg is present